tests/eval/basic.rs

```diff
@@ -0,0 +1,20 @@
+use crate::eval::eval_code;
+
+#[test]
+fn eval_print() {
+    assert_eq!(eval_code("print! 1").stdout, "1\n");
+    assert!(!eval_code("print 0").stderr.is_empty());
+    assert_eq!(eval_code("print! \"abc\"").stdout, "abc\n");
+    assert_eq!(eval_code("print! \"0.3\"").stdout, "0.3\n");
+    assert_eq!(eval_code("num = -3\nprint! num * 2").stdout, "-6\n");
+    assert_eq!(eval_code("print True").status.code().unwrap(), 1);
```

Comment:
Could you add a multi-line string with  `\"\"\"` and embedding expressions string with `\"\\{}\"`

Reply:
updated